src/apiCalls/cart/addToCart.js

```diff
@@ -0,0 +1,22 @@
+import axios from 'axios'
+import { Toast } from '../../components'
+
+export const addToCart = async (productDetails, cartDispatch, cartItems) => {
+  try {
+    const config = {
+      headers: {
+        authorization: localStorage.getItem('encodedToken'),
+      },
+    }
+    const response = await axios.post(
+      '/api/user/cart',
+      { product: { ...productDetails, qty: 1 } },

```

Comment:
You do not have to add qty: 1 separately, its getting added from backend only

Reply:
Thanks for pointing out this. I was unaware of it.